.gitlab/image.yml

```diff
@@ -1,8 +1,8 @@
 .use_status-job-settings:
   image: ginkgohub/cpu:openmpi-gnu9-llvm8
   tags:
-    - private_ci
-    - status-jobs
+    - cpu
+    - tum
```

Comment:
I set up a separate VM for status jobs, you can use the tag `status-jobs` for this, it shouldn't run on slurm.

Reply:
moved